Classes and objects

DZemitis opened the pull request.

DaveBarada committed:
Merge branch 'Arrays'

DaveBarada committed:
init

DaveBarada committed:
Exercise_1 done

DaveBarada committed:
init ex2

DaveBarada committed:
fixes

DaveBarada committed:
fix for Exercise_1

DaveBarada committed:
fix for FuelConsumptionCalculator

DaveBarada committed:
Exercise 4 done

DaveBarada committed:
energy drinks

DaveBarada committed:
ex5

DaveBarada committed:
ex 3

DaveBarada committed:
ex 7

DaveBarada committed:
ex1

DaveBarada committed:
Exercise 8 done

DaveBarada committed:
BankAccount done

DaveBarada committed:
exe 9 init

DaveBarada committed:
exe 8 done

DaveBarada committed:
ex Exercise_9 done

DaveBarada committed:
done

DaveBarada committed:
Account done

DaveBarada committed:
done

DaveBarada committed:
redone ex 3

DaveBarada committed:
continue

DaveBarada committed:
experiment

DaveBarada committed:
exe 3 final

DaveBarada committed:
fix

DaveBarada committed:
fix

ausmoons commented on csharp-basics/exercises/ClassesAndObjects/EnergyDrinks/Program.cs:
```diff
@@ -12,24 +8,26 @@ class Program
-             */
+            var energyDrinkers = CalculateEnergyDrinkers(NumberedSurveyed, PurchasedEnergyDrinks);
+
+
```
Pietiek ar vienu tukšu rindu, ja gribi atdalīt

ausmoons commented on csharp-basics/exercises/ClassesAndObjects/Exercise 1/Product.cs:
```diff
@@ -0,0 +1,21 @@
+﻿namespace Exercise_1
+{
+
```
Lieka tukša rinda

ausmoons commented on csharp-basics/exercises/ClassesAndObjects/Exercise 1/Product.cs:
```diff
@@ -0,0 +1,21 @@
+            PriceAtStart = price;
+            AmountAtStart = amount;
+        }
+        private string _name { get; set; }
```
Private nav { get; set; }

ausmoons commented on csharp-basics/exercises/ClassesAndObjects/Exercise 1/Program.cs:
```diff
@@ -0,0 +1,29 @@
+                product3,
+                product4
+            };
+            foreach (var prod in products)
```
Vajag tukšu rindu pēc }

ausmoons commented on csharp-basics/exercises/ClassesAndObjects/Exercise 3/Program.cs:
```diff
@@ -0,0 +1,11 @@
+    {
+        static void Main(string[] args)
+
+        {
```
?

DZemitis replied:
es biju uztaisījis trešajam uzdevumam jaunu failu, un sanāca gan 3, gan 3a, izdzēsu no VS, bet lokāli man viņš laikam vēl palika, es viņu tagad izdzēsu, un tad sanāk paliek tikai 3a, piedošan, es nesapratu kā īsti tas strādā, tagad, laikam, skaidrība.

ausmoons commented on csharp-basics/exercises/ClassesAndObjects/Exercise 3a/FuelGauge.cs:
```diff
@@ -0,0 +1,33 @@
+        public FuelGauge(int liters)
+        {
+            CurrentLiters = liters;
+        }
```
Vajag tukšu rindu pēc }

DaveBarada committed:
Exercise_1 dzēsta tukša rinda

Exercise_1 noņemts privātajam mainīgajam {get;set;}
Exercise_1 pielikt tukša rinda pēc foreach
izdzēsts Exercise 3, jo 3a ir final
Exercise_3a ievietota tukša rinda pēc klases metodes
